operate/client/src/modules/validators/index.ts

```diff
@@ -248,23 +249,36 @@ const validateMultipleVariableValuesValid: FieldValidator<
   return ERRORS.variables.multipleValueInvalid;
 }, VALIDATION_TIMEOUT);
 
+/**
+ * Validates if value contains only characters from a key or UUID
+ */
 const validateOperationIdCharacters: FieldValidator<
   ProcessInstanceFilters['operationId']
 > = (value = '') => {
-  if (value !== '' && !/^[a-f0-9-]{1,36}/.test(value)) {
+  const schema = z.union([
+    z.string().length(0),
+    z.string().regex(/^[0-9]+$/),
+    z.string().regex(/^[a-f0-9-]{1,36}/),
+  ]);
```

Comment:
I'm wondering if we should move the definition to outside the function so we don't create a new schema everytime